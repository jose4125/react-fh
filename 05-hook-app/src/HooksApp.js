import { CounterApp } from "./components/01-useState/CounterApp";
import { CounterWithCustomHook } from "./components/01-useState/CounterWithCustomHook";
import { SimpleForm } from "./components/02-useEfect/SimpleForm";
import { FormWithCustomHooks } from "./components/02-useEfect/FormWithCustomHook";
import { MultipleCustomHooks } from "./components/03-examples/MultipleCustomHooks";
import { FocusScreen } from "./components/04-useRef/FocusScreen";
import { Memorize } from "./components/06-memos/Memorize";
import { MemoHook } from "./components/06-memos/MemoHook";
import { CallbackHook } from "./components/06-memos/CallbackHook";
import { Padre } from "./components/07-tarea-memo/Padre";
import { TodoApp } from "./components/08-useReducer/TodoApp";

function HooksApp() {
  return (
    <div className="App">
      <CounterApp />
      <CounterWithCustomHook />
      <SimpleForm />
      <FormWithCustomHooks />
      <MultipleCustomHooks />
      <FocusScreen />
      <Memorize />
      <MemoHook />
      <CallbackHook />
      <Padre />
      <TodoApp />
      <br />
      <br />
      <br />
      <br />
    </div>
  );
}

export default HooksApp;
