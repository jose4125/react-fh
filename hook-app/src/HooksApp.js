import { CounterApp } from "./components/01-useState/CounterApp";
import { CounterWithCustomHook } from "./components/01-useState/CounterWithCustomHook";
import { SimpleForm } from "./components/02-useEfect/SimpleForm";
import { FormWithCustomHooks } from "./components/02-useEfect/FormWithCustomHook";
import { MultipleCustomHooks } from "./components/03-examples/MultipleCustomHooks";
import { FocusScreen } from "./components/04-useRef/FocusScreen";

function HooksApp() {
  return (
    <div className="App">
      <CounterApp />
      <CounterWithCustomHook />
      <SimpleForm />
      <FormWithCustomHooks />
      <MultipleCustomHooks />
      <FocusScreen />
    </div>
  );
}

export default HooksApp;
