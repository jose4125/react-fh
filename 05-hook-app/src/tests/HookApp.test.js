import { shallow } from "enzyme";
import HooksApp from "../HooksApp";

describe("test HooksApp", () => {
  test("Should render properly", () => {
    const wrapper = shallow(<HooksApp />);
    expect(wrapper).toMatchSnapshot();
  });
});
