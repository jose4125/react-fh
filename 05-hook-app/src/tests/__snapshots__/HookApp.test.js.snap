// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test HooksApp Should render properly 1`] = `
<div
  className="App"
>
  <CounterApp />
  <CounterWithCustomHook />
  <SimpleForm />
  <FormWithCustomHooks />
  <MultipleCustomHooks />
  <FocusScreen />
  <Memorize />
  <MemoHook />
  <CallbackHook />
  <Padre />
  <TodoApp />
  <br />
  <br />
  <br />
  <br />
</div>
`;
